src/table/paginator.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 100%;
+    flex: 1;
+  }
+
+  > .iui-center {
+    width: 33%;
+    justify-content: center;
+    @include themed {
+      color: t(iui-text-color);
+    }
+
+    .iui-button-group {
```

Comment:
I guess button-group is fine but I think we should modify the "active" stripe to show below the number.

Reply:
I'm cool with that.  Done in the next push.
![Screen Shot 2021-05-25 at 3 07 42 PM](https://user-images.githubusercontent.com/849817/119554682-f6384d00-bd6a-11eb-9066-a52e0d879604.png)